### ISSUE TITLE
[Accessibility]: Command palette doesn't look good at mobile sizes

### ISSUE THREAD

**Issue by tjkandala** [sourcegraph/sourcegraph#36861]
### Audit type

Mobile device navigation

### User journey audit issue

#34061

### Problem description

When opening the command palette on mobile screen sizes, the rest of the site shrinks?

### Expected behavior

The command palette should adapt to mobile screen sizes.

### Additional details

![Screen Shot 2022-06-08 at 4 24 12 PM](https://user-images.githubusercontent.com/37420160/172710267-e4427d53-ea1b-48b6-a61d-b57845fd0957.png)

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.

**Comment by umpox** [comment]
Closed by https://github.com/sourcegraph/sourcegraph/pull/37523
